### ISSUE TITLE
Development version instead of actual version in docker image

### ISSUE THREAD

**Issue by j-fuentes** [jetstack/preflight#23]
The binary in the docker image does display the version information.

**What happened**

```
$ docker run quay.io/jetstack/preflight:v0.1.9 version -v
Preflight version:  development

Commit:
Built:
Go:
```

**What should happen**

```
$ docker run quay.io/jetstack/preflight:v0.1.9 version -v
Preflight version:  v0.1.9

Commit: <commit hash>
Built: <build timestamp>
Go: <Go version>
```

**Comment by paulwilljones** [comment]
Closed by #39

**Comment by paulwilljones** [comment]
Resolved by #39

**Comment by paulwilljones** [comment]
@j-fuentes this should have been closed as part of #39
